maps/scrape.py

```diff
@@ -16,25 +17,34 @@ def scrape_calls(date1: datetime, date2: datetime):
     return j
 
 
-def insert_data(calls):
-    with sqlite3.connect(DB_PATH) as conn:
-        c = conn.cursor()
-        prepared = []
-        for call in calls:
-            dt = datetime.strptime(call["DispatchTime"] + " " + call["DispatchTime2"], "%m-%d-%Y %H:%M:%S")
-            values = (dt, call["lat"], call["lon"], call["City"], call["CallType"], call["Address"])
-            prepared.append(values)
-        c.executemany('''INSERT OR REPLACE INTO calls (datetime, lat, lon, city, call_type, address)
-        VALUES (?,?,?,?,?,?)''', prepared)
-
-    conn.commit()
-
-
-def fetch_data(days_ago=1):
+def insert_data(call_data, v=0):
+    for row_data in call_data:
+        call = None
+        try:
+            call = Call(row_data)
+        except Exception as e:
+            app.logger.warning('Scraper: {}'.format(e))
+        if call:
+            existing_id = CallQuery.get_existing_id(call)
+            if existing_id is not None:
+                call.id = existing_id
+                if v >= 2:
+                    app.logger.info('Scraper: Merging {} into existing Call'.format(call))
```

Comment:
Needs a silent mode for production, or only log in case of error

Reply:
And what is V?

Reply:
It will stop logging info when v=0, or you can set the logger to be silent somewhere in __init__.py (when it defines the level of logging https://github.com/katzrkool/maps/blob/1940eebe7a67c8af7018696c9611d136de45ed9e/maps/__init__.py#L22